packages/google_sign_in/google_sign_in_ios/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## Next
+
+* Remove the requirement for a `GoogleService-Info` file.
```

Comment:
Can we have a more detailed description here?

Reply:
Done